widgetssdk/src/main/res/values/donottranslate.xml

```diff
@@ -2,4 +2,16 @@
 <resources>
     <string name="glia_outlined_options_icon_content_description">Outlined options icon description</string>
     <string name="glia_file_preview_transition_name">image_preview_transition</string>
+
+    <!-- Survey -->
+    <string name="glia_survey_open_text_item_title" translatable="false">Additional thoughts?</string>
+    <string name="glia_survey_scale_item_title" translatable="false">How would you rate this interaction? (1 - Bad, 5 - Good)</string>
+    <string name="glia_survey_scale_item_1" translatable="false">1</string>
+    <string name="glia_survey_scale_item_2" translatable="false">2</string>
+    <string name="glia_survey_scale_item_3" translatable="false">3</string>
+    <string name="glia_survey_scale_item_4" translatable="false">4</string>
+    <string name="glia_survey_scale_item_5" translatable="false">5</string>
+    <string name="glia_survey_single_choice_item_title" translatable="false">If you hadn\'t chatted with us today, what would you have done?</string>
```

Comment:
I think that survey items can or should also be translated (except 1-5 numbers).

Reply:
I think that numbers actually might be also available for translations. Very long ago worked on an app that needed that. 
![image](https://github.com/salemove/android-sdk-widgets/assets/10000880/1683a770-8c6b-4957-b1c7-b7aa016e098a)
But probably we won't need that anytime soon so maybe it is not important )

Reply:
Probably you can ignore my previous comment

Reply:
These seem to be some temporary implementation. Waiting for final confirmation from Davit

Reply:
I'm sorry, I forgot to check(((
But at first glance, it looks like these strings are placeholders and should be replaced with text from the server. However, on IOS there are no such texts as far as I'm informed, @Pelkar can confirm.
As far as I remember, @andrews-moc implemented Survey, so he knows better than me.

Reply:
Yes, iOS has nothing like this

Reply:
Sorry for a late reply. I double-checked and indeed, looks like these survey strings should not be displayed. They are just like placeholders and there shouldn't be a case when they are displayed.

@AndriiHorishniiMOC Please confirm if you remember because if I'm not mistaken, you've added them initially.

Reply:
@AndriiHorishniiMOC any input?

Reply:
Strings `glia_survey_scale_item_1` ... `glia_survey_scale_item_5` are used as labels for the buttons. In my view, they can be translated.

`glia_survey_open_text_item_title`, `glia_survey_scale_item_title`, `glia_survey_single_choice_item_title` - better to remove them, cause those values are replaced with text from the server.